.github/workflows/main.yml

```diff
@@ -23,270 +23,82 @@ on:
-      run: dotnet sonarscanner end /d:sonar.login="${{ secrets.SONAR_TOKEN }}"
-
-  build-package:
-    name: Build Package
-    outputs:
-        taggedVersion: ${{ steps.image.outputs.taggedVersion }}
-    # The type of runner that the job will run on
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v2
-      with:
-        fetch-depth: 1
-
     - name: Get Image Version
       uses: VirtoCommerce/vc-github-actions/get-image-version@dev
       id: image
 
-    - name: Install VirtoCommerce.GlobalTool
-      run: |
-        dotnet tool install --global VirtoCommerce.GlobalTool
+    - name: Get changelog
+      id: changelog
+      uses: VirtoCommerce/vc-github-actions/changelog-generator@dev
 
-    - name: Build Package
-      run: vc-build Compress -skip Test
+    - name: Add version suffix
+      uses: VirtoCommerce/vc-github-actions/add-version-suffix@dev
 
-    - name: Build Docker Image
-      shell: pwsh
-      run: |
-        Invoke-WebRequest -Uri https://raw.githubusercontent.com/VirtoCommerce/vc-docker/master/linux/platform/Dockerfile -OutFile artifacts/Dockerfile
-        Invoke-WebRequest -Uri https://github.com/VirtoCommerce/vc-docker/blob/master/linux/platform/wait-for-it.sh -OutFile artifacts/wait-for-it.sh
-        docker build artifacts --build-arg SOURCE=. --tag "docker.pkg.github.com/$('${{ github.repository }}'.ToLower())/platform:${{ steps.image.outputs.taggedVersion }}"
-    - name: Docker Login
-      uses: azure/docker-login@v1
-      with:
-        login-server: docker.pkg.github.com
-        username: $GITHUB_ACTOR
-        password: ${{ secrets.GITHUB_TOKEN }}
-
-    - name: Upload a Build Artifact
-      uses: actions/upload-artifact@v2
-      with:
-        # Artifact name
-        name: Platform # optional
-        # A file, directory or wildcard pattern that describes what to upload
-        path: artifacts/publish/  
+    - name: SonarCloud Begin
+      uses: VirtoCommerce/vc-github-actions/sonar-scanner-begin@dev
 
-    - name: Push Docker Image to GitHub
-      run: |
-        REPOSITORY=$(echo $GITHUB_REPOSITORY | tr '[A-Z]' '[a-z]') 
-        docker push "docker.pkg.github.com/$REPOSITORY/platform:${{ steps.image.outputs.taggedVersion }}"
-  
-  swagger-validation:
-    name: Swagger Schema Validation
-    # The type of runner that the job will run on
-    runs-on: ubuntu-latest
-    needs: build-package
-    steps:
-    - uses: actions/checkout@v2
+    - name: Build
+      run: vc-build Compile
 
-    - name: Install VirtoCommerce.GlobalTool
-      run: dotnet tool install --global VirtoCommerce.GlobalTool
+    - name: Unit Tests
+      run: vc-build Test -skip
 
-    - name: Download a Build Artifact
-      uses: actions/download-artifact@v2
+    - name: SonarCloud End
+      uses: VirtoCommerce/vc-github-actions/sonar-scanner-end@dev
+
+    - name: Packaging
+      run: vc-build Compress -skip Clean+Restore+Compile+Test
+
+    #- name: Swagger Schema Validation
```

Comment:
Is it OK that Swagger Schema Validation is commented out?

Reply:
It is temporary  because of https://virtocommerce.atlassian.net/browse/VP-4255